frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx

```diff
@@ -310,26 +269,7 @@ const ErrorInstance: React.FC<Props> = ({ errorGroup }) => {
 						>
 							<KeyboardShortcut label="Next" shortcut="]" />
 						</Tooltip>
-						<LinkButton
-							kind="primary"
-							emphasis="high"
-							to={getLogsLink(errorInstance.error_object)}
-							disabled={
-								!isLoggedIn ||
-								!errorInstance.error_object.session
-							}
-							trackingId="error-related_logs_link"
-						>
-							<Box
-								display="flex"
-								alignItems="center"
-								flexDirection="row"
-								gap="4"
-							>
-								<IconSolidLogs />
-								Show logs
-							</Box>
-						</LinkButton>
+						<ShowLogsButton data={data} />
```

Comment:
Moved the logic into a separate component because `ErrorInstance` is quite big and it's easier to wrangle this if it's in a separate file.